config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/environment/FullBundleCreator.java

```diff
@@ -82,18 +95,27 @@ public void createFullBundle(Map<String, String> environmentProperties,
 
         // process for reattaching loose encasses and write to the final path
         dependencyBundlesProcessor.process(singletonList(fullBundleFile), bundleFolderPath);
+        int index = bundleFileName.indexOf(BUNDLE_EXTENSION);
+        documentFileUtils.createFile(elementPair.getRight(), new File(bundleFolderPath,
+                bundleFileName.substring(0, index) + DELETE_BUNDLE_EXTENSION).toPath());
```

Comment:
change this to bundleEnvironmentValues.getLeft() + DELETE_BUNDLE_EXTENSION

Reply:
full bundle name is bundleEnvironmentValues.getLeft() + "." + configName.get() + ".full.bundle";

so i am just replacing .bundle with .delete.bundle

Reply:
just a suggestion, It is good to compute from bundleEnvironmentValues.getLeft() , instead of depending on full install bundle filename

Reply:
anyway this will change with nikhil's naming pr